flink-runtime/src/test/java/org/apache/flink/runtime/io/disk/FileChannelManagerImplTest.java

```diff
@@ -89,28 +90,29 @@ public void testFairness() throws Exception {
             threads[i].join();
         }
 
-        assertEquals(counter1.get(), counter2.get());
+        assertThat(counter2.get()).isEqualTo(counter1.get());
```

Comment:
```suggestion
        assertThat(counter2).isEqualTo(counter1);
```

Reply:
Sorry, I got it now.

How about `assertThat(counter2).hasValue(counter1.get());`? 

nit: This is a really nitty remark. Feel free to ignore it.